.github/pull_request_template.md

```diff
@@ -3,6 +3,20 @@
  <!-- please add an icon to the title of this PR (see ../CONTRIBUTING.md) -->
  <!-- the icon will be either ⚠️ (:warning:, major or breaking changes), ✨ (:sparkles:, feature additions), 🐛 (:bug:, patch and bugfixes), 📖 (:book:, documentation or proposals), 💚 (`:green_heart:`, testing), 💎 (`:gem:`, refactor), 🔧 (`:wrench:`, dev tooling and chores) or 🌱 (:seedling:, minor or other) -->
 
+
+**What type of PR is this?**
+
+<!--
+Add one of the following kinds:
+/kind bug
+/kind documentation
+/kind feature
+/kind testing
+/kind refactor
+/kind dev-tooling
+/kind other
+-->
+
```

Comment:
- are these reasonable types?
 - do we still need the icons?

Reply:
This might be an unpopular point of view, but I don't particularly like the icons. They are kind of a pain. I always try to add the `:sparkles:` or other to the title and then remember I have to go find the icon somewhere else and copy/paste it into the title b/c the text doesn't transform to the icon (etc...). I prefer the labels.

One label that may be missing is `/kind breaking-change`. I think we'll want to call these out specifically.

Perhaps, `/kind breaking-change` should be more like `/label breaking-change` as it would be a modifier of a given `/kind`.

Reply:
I've actually heard that from a few folks, you are not the only one that doesn't like the icons :D

Reply:
+1 for labels, they are easier to add than emojis and can be filtered on more easily.

For breaking changes we already have the "action required" keyword in the release note that marks it as breaking in the release. There is also a label "release-note-action-required" which may or may not get applied automatically when that keyword is used in the PR description release note.

Reply:
if we agree on the labels, I can make the other changes

> There is also a label "release-note-action-required" which may or may not get applied automatically when that keyword is used in the PR description release note.

yes, it does get applied by the release-notes tool

Reply:
updated